src/client/Client.cc

```diff
@@ -10679,6 +10679,7 @@ int Client::_do_setxattr(Inode *in, const char *name, const void *value,
   req->head.args.setxattr.flags = xattr_flags;
 
   bufferlist bl;
+  assert (value != NULL || size != 0);
```

Comment:
@batrick Yeah correct. I believe assert (value != NULL && size != 0); would be good?